src/app/containers/Byline/index.jsx

```diff
@@ -13,16 +13,15 @@ const BylineContainer = ({ blocks, className }) => {
 
   const { name, title } = blocks[0];
 
-  if (name && title) {
-    return (
-      <GridItemMedium className={className}>
-        <Byline service={service} name={name} title={title} />
-      </GridItemMedium>
-    );
-  }
-  return null;
+  return (
+    <GridItemMedium className={className}>
+      <Byline service={service} name={name} title={title} />
+    </GridItemMedium>
+  );
 };
 
-BylineContainer.propTypes = bylineBlockPropTypes;
+BylineContainer.propTypes = {
+  ...bylineBlockPropTypes,
```

Comment:
I'd probably modify these prop types to make the `name` and `title` required rather than optional as they are at the moment